pkg/runner/cluster_swarm.go

```diff
@@ -133,8 +133,20 @@ func (*ClusterSwarmRunner) Run(ctx context.Context, input *api.RunInput, ow io.W
 		return nil, fmt.Errorf("testground-redis service doesn't exist in the swarm cluster; aborting")
 	}
 
+	// We can't create a network for every testplan on the same range,
+	// so we check how many networks we have and decide based on this number
```

Comment:
* Can we choose based on how many _testground_ networks we have?
* Can we allocate from `16.0.0.0/4`? That gives us 16.0.0.0 -> 31.0.0.0.
  * Public IP addresses.
  * We're not going to run out after 256 runs.

Reply:
@raulk @Stebalien given that we run only one testplan at a time, do we actually need multiple testplan networks? If we start enforcing the `one concurrent testplan run`, and make sure we clean up after the testplan, we could simplify this.

Reply:
@Stebalien I am confused about what you're asking - `16.0.0.0 -> 31.0.0.0` - isn't this public space? Nevertheless pushed a commit with it.

Reply:
> @raulk @Stebalien given that we run only one testplan at a time, do we actually need multiple testplan networks? If we start enforcing the one concurrent testplan run, and make sure we clean up after the testplan, we could simplify this.

Well, I think the goal was to be able to run multiple at the same time.

> @Stebalien I am confused about what you're asking - 16.0.0.0 -> 31.0.0.0 - isn't this public space? Nevertheless pushed a commit with it.

Yes. Libp2p may choose to behave differently when it doesn't have a publicly routable IP address. Ideally, every instance would have two interfaces with two IP addresses (and would be able to turn them on/off with the sidecar).